code/model/DMSDocument.php

```diff
@@ -914,6 +923,13 @@ public function getCMSFields()
         $fields->add(new TextField('Title', 'Title'));
         $fields->add(new TextareaField('Description', 'Description'));
 
+        $downloadBehaviorSource = array(
+            '' => 'Default ('.Config::inst()->get('DMSDocument', 'default_download_behaviour').')',
```

Comment:
I hear you, and now understand where you're coming from.

What about the case where you want to revert a Document to the global setting? And then you want to change the global setting? Would this not require manually re-defaulting every recently updated Document?